transaction/builder/src/lib.rs

```diff
@@ -3,9 +3,14 @@
 //! Utilities for creating MobileCoin transactions, intended for client-side
 //! use and not intended to be used inside of enclaves.
 
+#![no_std]
 #![deny(missing_docs)]
 
-extern crate core;
+extern crate alloc;
+
+// This helps the `maplit` crate macros to work
+#[cfg(test)]
+extern crate std;
```

Comment:
I think you an do 
```
#![cfg_attr(not(test), no_std)]
#![deny(missing_docs)]

extern crate alloc;

```

Reply:
I guess I have a slight preference the other way, just because I find it simpler to declare conditional dependencies than to declare conditional negative dependencies.

But it's ultimately the same, and I think a lot of James' crates do it the way you showed.